src/BSeries.jl

```diff
@@ -1886,9 +1899,9 @@ end
 # of coefficients.
 # Checks whether `trees` and `ocefficients` satisfify the energy_preserving
 # condition.
-function _is_energy_preserving(trees, coefficients)
+function _is_energy_preserving(trees, coefficients, atol, rtol)
     # TODO: `Float32` would also be nice to have. However, the default tolerance
-    #       of the rank computation is based on `Float64`. Thus, it will usually
+    #       of the rank computation is based on `Real`. Thus, it will usually
```

Comment:
Why did you change this?

Reply:
So that it accepts both `Int` and `Float64`.

Reply:
Ah, sorry. I'll fix it back.